### ISSUE TITLE
Display launcher last release version

### ISSUE THREAD

**Issue by Herklos** [Drakkar-Software/OctoBot-Launcher#3]

**Comment by PKBO** [comment]
launcher not working, octobot itself not working, even on linux or windows. 

On windows it simply doesnt open, on linux nothink happen when you double click on it, and if i use launcher, i get some error about missing somethink, than octobot is not possible to open again

TIME: 'REAL_TIME'>: {}, <EvaluatorMatrixTypes.STRATEGIES: 'STRATEGIES'>: {'FullMixedStrategiesEvaluator': -0.0423563180676845}}
XPCOMGlueLoad error for file /usr/lib/firefox/libmozgtk.so:
/tmp/_MEId7Ohcl/libstdc++.so.6: version `CXXABI_1.3.8' not found (required by /usr/lib/x86_64-linux-gnu/libmirclient.so.9)
Couldn't load XPCOM.

**Comment by Herklos** [comment]
Hi PKBO,

Thank you for your possible bug report.
Nice to hear that windows is now working.

On linux, it seems to be a problem with firefox, not OctoBot, can you try to start firefox ?
`$ firefox`

**Comment by PKBO** [comment]
Thanks, your right, it runs on localhost when opened in firefox manualy. 

Is there some Discord or Telegram chanel for comunity ? 

And how about staggered order, is it possible to use them somehow ? iam not interested in only trading, instead i want to try "market making" orders, like DEXbot has for bts markets.

**Comment by Herklos** [comment]
Yes you can find links to the recently created OctoBot community [here](https://github.com/Drakkar-Software/OctoBot#octobot-community).

Currently OctoBot has no staggered orders strategy. As OctoBot is fully modular (with its tentacles that you can install) you can develop or find someone that developed this strategy and install & use it.
